README.md

```diff
@@ -1,6 +1,8 @@
 # gcloud-kubectl-helm
 Docker image for the quaternity of [gcloud](https://cloud.google.com/sdk/docs/), [helm](https://www.helm.sh), [kubectl](https://kubernetes.io/docs/reference/kubectl/kubectl/) and [SOPS](https://github.com/mozilla/sops).
 
+The image also contains [cloud_sql_proxy](https://github.com/GoogleCloudPlatform/cloudsql-proxy), [gnupg](https://pkgs.alpinelinux.org/package/v3.3/main/x86/gnupg), [mysql-client](https://pkgs.alpinelinux.org/package/edge/main/x86/mysql-client) and [yq](https://github.com/mikefarah/yq).
```

Comment:
Can we guarantee that mysql is installed in 3.3?
Because you specify no version when calling `apk add`

Reply:
changed link to gnupg, without version...